src/lucky_flow/element.cr

```diff
@@ -1,7 +1,7 @@
 class LuckyFlow::Element
   private getter raw_selector
   getter inner_text
-  delegate text, click, send_keys, displayed?, attribute, to: element
+  delegate text, click, send_keys, displayed?, attribute, property, to: element
```

Comment:
while `attribute` is no longer used in the class, I wasn't sure about removing it from this list since it is technically public API. I can remove it, though

Reply:
I think we should keep it in. At least for this next release. We can see about deprecating it later.